server/src/main/java/pl/edu/agh/server/term/TermService.java

```diff
@@ -0,0 +1,30 @@
+package pl.edu.agh.server.term;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+
+@Component
+public class TermService {
+    private final TermRepository termRepository;
+
+    @Autowired
```

Comment:
ta adnotacja i konstruktor jest nie potrzebne, jak usunieesz cały i dodasz nad klasą adnotacje @RequiredArgsConstructor to sie samo sklei